## IMPORT AND CREATE DATAFRAME

In [1]:
#Import Libraries
import pandas as pd
import numpy as np
import scipy.stats as stats

In [ ]:
#Create DataFrame with 1 coloumn and 25 rows
df = pd.DataFrame([100, 150, 50, 100, 130, 120,100, 85, 70, 150, 150,
                120, 50, 100, 100, 140, 90, 150, 50, 90, 120, 100, 110, 75, 65],
                columns = ['Transaction Amount'])
df

Transaction Amount
0                  100
1                  150
2                   50
3                  100
4                  130
5                  120
6                  100
7                   85
8                   70
9                  150
10                 150
11                 120
12                  50
13                 100
14                 100
15                 140
16                  90
17                 150
18                  50
19                  90
20                 120
21                 100
22                 110
23                  75
24                  65

## MEASURE OF CENTRAL TENDENCY

In [ ]:
#Central Tendency
import statistics
print(statistics.mean(df['Transaction Amount']))
print(statistics.median(df['Transaction Amount']))
print(statistics.mode(df['Transaction Amount']))

102.6
100
100


Based on the calculation of the measure of central tendency, the following results were obtained:
1. The mean is 102.6.
2. The median (Central Value / Quartile 2) is 100.
3. The mode, which is the most frequent value, is 100.

Therefore, it can be concluded that the data tends to cluster around the value of 100.

## MEASURE OF VARIABILITY


In [ ]:
#Variability
import statistics
print(np.var(df['Transaction Amount']))
print(np.std(df['Transaction Amount']))
print(np.max(df['Transaction Amount']))
print(np.min(df['Transaction Amount']))
print(np.max(df['Transaction Amount']) - np.min(df['Transaction Amount']))
print(np.quantile(df['Transaction Amount'], 0.25))
print(np.quantile(df['Transaction Amount'], 0.5))
print(np.quantile(df['Transaction Amount'], 0.75))

972.2399999999997
31.180763300471007
150
50
100
85.0
100.0
120.0


Based on the calculation of the measure of variability, the following results were obtained:
1. Variance of 972.24
2. Standard deviation of 31.18
3. Maximum value of 150
4. Minimum value of 50
5. Range (Difference between Max and Min) of 100
6. Quartile 1 (Q1) of 85
7. Quartile 2 (Q2) of 100
8. Quartile 3 (Q3) of 120

The data shows that the spread of values is quite broad, with a range of 100 and a standard deviation of 31.18, indicating considerable variation around the mean.

## PROSES T-TEST

In [ ]:
#Define H0 and H1
#H0: The mean transaction amount from 25 salespeople after training is = 100
#H1: The mean transaction amount from 25 salespeople after training is > 100

In [ ]:
#Define alpha = 0.05

In [ ]:
#Calculate t statistics and P-value
#Perform one sample t-test

import scipy.stats as stats

In [ ]:
len (df)

25

In [ ]:
stats.ttest_1samp(a=df, popmean=100)

TtestResult(statistic=array([0.40850016]), pvalue=array([0.68652848]), df=array([24]))

In [ ]:
#statistic= 0.40850016, pvalue= 0.68652848

**Result t-statistic = 0.40850016, p-value = 0.68652848**

Since the p-value is greater than the significance level alpha (p-value = 0.69 > alpha = 0.05), we fail to reject H0.
Therefore, there is not enough evidence to suggest that the average transaction amount of the 25 salespeople after the training is greater than 100.

In [ ]:
#Conclude with t statistics and critical region
#Find the critical region

In [ ]:
stats.t.ppf(1-0.05, 29)

np.float64(1.6991270265334972)

**Result Critical Region = 1.6991270265334972**

Since the t-statistic value is smaller than the critical region (t-statistic = 0.41 < Critical Region = 1.70), we fail to reject H0.
Therefore, there is not enough evidence to claim that the average transaction amount of the 25 salespeople after the training is greater than 100.

## CONCLUSION

Based on both statistical tests, we fail to reject H₀, which means there is not enough evidence to suggest that the average transaction amount of the 25 salespeople after the training is greater than 100.